nuxeo-opencmis-tests/src/test/java/org/nuxeo/ecm/core/opencmis/impl/TestCmisBinding.java

```diff
@@ -2264,6 +2266,48 @@ public void testQueryContains() throws Exception {
         }
     }
 
+    /**
+     * Test the relax mode having multiple {@code CONTAINS()}s in CMISQL. The relax mode does not follow the CMIS
+     * specification 1.1 where at most one {@code CONTAINS()} function MUST be included in a single query statement
+     * (Spec 2.1.14.2.4.4). This relax mode only work for CMISQL query having no {@code JOIN} predicate.
+     *
+     * @throws Exception
+     * @see https://jira.nuxeo.com/browse/NXP-19858
+     */
+    @Test
+    @LocalDeploy("org.nuxeo.ecm.core.opencmis.tests.tests:OSGI-INF/test-unfollow-cmis-spec.xml")
+    public void testQueryMultiContains() throws Exception {
+
+        if (supportsJoins()) {
+            return;
+        }
+
+        final ConfigurationService configService = Framework.getService(ConfigurationService.class);
```

Comment:
Yes it's clear it won't be changed (also this is test code). Even in non-test code we don't add `final` to variables or parameters when it's not required by the compiler, as it brings nothing much to the compiler (which can infer it) or the reader (who sees just an extra spurious keyword).